jsign-core/src/main/java/net/jsign/mscab/CFHeader.java

```diff
@@ -177,7 +177,9 @@ public void headerDigestUpdate(MessageDigest digest) {
 
         if (this.abReserved != null) {
             if (this.abReserved.length > RESERVE_CNT_HDR_LEN) {
-                int cbJunk = this.abReserved[0] | ((((int)this.abReserved[1] & 0xff) << 8));
```

Comment:
this.abReserved[0] | ((((int)this.abReserved[1] & 0xff) << 8));
is not correct. because 0xff is recognized as -1.

(this.abReserved[0] & 0xff) | ((((int)this.abReserved[1] & 0xff) << 8));
is correct.

And I used ByteBuffer instead of formula for clean code.